<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoPlayClass</class>
 <widget class="QWidget" name="DemoPlayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DemoPlay</string>
  </property>
  <widget class="JVideoWidget" name="video">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="JSlider" name="playPos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>1231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::add-page{   
    background-color: rgb(87, 97, 106);
	height:4px;
}
QSlider::sub-page {
    background-color: rgb(37, 168, 198);
	height:4px;
}
QSlider::groove {
	background:transparent;
	height:6px;
	border-radius:5px;
}
QSlider::handle {
    height: 30px;
    width:13px;
    border-image: url(:/DemoPlay/Resources/ic_volume_thumb.png);
    margin: -4 0px; 
}</string>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="isPlay">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>JVideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>JVideoWidget.h</header>
  </customwidget>
  <customwidget>
   <class>JSlider</class>
   <extends>QSlider</extends>
   <header>JSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DemoPlay.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DemoPlayClass</receiver>
   <slot>OpenFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>642</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>762</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isPlay</sender>
   <signal>clicked()</signal>
   <receiver>DemoPlayClass</receiver>
   <slot>PlayOrPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>758</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playPos</sender>
   <signal>sliderPressed()</signal>
   <receiver>DemoPlayClass</receiver>
   <slot>SliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>702</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>761</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playPos</sender>
   <signal>sliderReleased()</signal>
   <receiver>DemoPlayClass</receiver>
   <slot>SliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>706</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>755</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenFile()</slot>
  <slot>PlayOrPause()</slot>
  <slot>SliderPress()</slot>
  <slot>SliderRelease()</slot>
 </slots>
</ui>
